spec/ddtrace/profiling/ext/forking_spec.rb

```diff
@@ -65,9 +67,9 @@
         #       The results of this will carry over into other tests...
         #       Just assert that the receiver was patched instead.
         #       Unfortunately means we can't test if "fork" works in main Object.
-        expect(toplevel_receiver)
-          .to receive(:extend)
-          .with(described_class::Kernel)
+        # expect(toplevel_receiver.class)
```

Comment:
This no longer occurs but I thought it worthwhile to leave commented here as a suggestion of what we were doing in case we have to revisit this kind of patching again.